docs/server/snippets/plugin-fieldLabel.js

```diff
@@ -1,11 +1,13 @@
-var plugin = function (config) {
-
+const plugin = () => {
   return {
-    fieldLabel: function (field) {
+    fieldLabel: function(field) {
```

Comment:
Any reason to not use an arrow function here?

Reply:
Nope, probably just copied over from somewhere else. Fixed. I'd like to go through the codebase and switch lots of things to arrow functions as well, but had to cut things off somewhere. 😄

Reply:
I get that!